docs/src/main/tut/user-guide.md

```diff
@@ -441,7 +441,7 @@ For all `String`-based endpoints, Finch provides an `as[A]` method to perform ty
 is available for any `Endpoint[String]`, `Endpoint[Option[String]]` or `Endpoint[Seq[String]]` as
 long as a matching implicit `io.finch.DecodeEntity[A]` type-class is in the scope.
 
-This facility is designed to be intuitive, meaning that you do not have to provide a
+This facility is designed to be intuitive, meaning that you do not have to provide an
```

Comment:
If you would like to go through and change places that are using full paths to describe specific paths, I think that would be good. We probably want to be consistent in the docs. In that case you linked, it is specifically talking about what package to import so we can't change that. If you just want to do the ones in this PR that's fine too, I'm happy to go through and change the rest at some point .